service/resources.ts

```diff
@@ -1,14 +1,44 @@
 import { Empty } from "google-protobuf/google/protobuf/empty_pb";
 import client from "service/client";
 
+export interface CommunityGuideline {
+  title: string;
+  guideline: string;
+  icon: string;
+}
+
 export async function getTermsOfService() {
   const res = await client.resources.getTermsOfService(new Empty());
   return res.toObject();
 }
 
-export async function getCommunityGuidelines() {
-  const res = await client.resources.getCommunityGuidelines(new Empty());
-  return res.toObject();
+export async function getCommunityGuidelines(): Promise<CommunityGuideline[]> {
+  return [
+    {
+      title: "Be kind to one another",
+      guideline:
+        "Couchers.org is an inclusive community where people of all cultures and backgrounds come together. Treat everyone with mutual respect and dignity.",
+      icon: "hand_with_heart.svg",
+    },
+    {
+      title: "This is not a dating app",
+      guideline:
+        "Couchers.org is NOT for dating or hooking up. Harassment or or using the platform for the purpose of dating or hooking up will result in a ban.",
+      icon: "ghost.svg",
+    },
+    {
+      title: "Be safe and sensible",
+      guideline:
+        "Read profiles, references, and resources. Report any inappropriate content or behavior.",
+      icon: "shield.svg",
+    },
+    {
+      title: "Keep money out of it",
+      guideline:
+        "Keep things non-transactional. Don't exchange money while staying or hosting. Couchers.org is about learning, experiences, and getting to know people.",
+      icon: "handshake.svg",
+    },
+  ];
 }
```

Comment:
I wonder if this is better suited to live in the frontend in one of the translation JSON file instead, where each version is linked to a different list of guidelines so we can use the version number retrieved from the backend to retrieve the right guidelines. After all, this feels like "static resource" to me where if we keep it at the frontend, we can also translate this to multiple languages which currently isn't possible since it comes from the backend.

Reply:
I'm not sure what the reason was for doing it this way in the first place
https://github.com/Couchers-org/couchers/issues/1682
I am indifferent on where it lives :P

Reply:
makes sense to me. changed in https://github.com/Couchers-org/web-frontend/pull/246/commits/0de5894fdac24e575f487d961b13f7d622218030